src/components/controls/toggle-focus.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { connect } from "react-redux";
+import { ImLab } from "react-icons/im";
+import { FaInfoCircle } from "react-icons/fa";
+import { Dispatch } from "@reduxjs/toolkit";
+import Toggle from "./toggle";
+import { SidebarIconContainer, StyledTooltip } from "./styles";
+import { TOGGLE_TREE_FOCUS } from "../../actions/types";
+import { RootState } from "../../store";
+
+
+function ToggleFocus({ tooltip, treeFocus, layout, dispatch, mobileDisplay }: {
+  tooltip: Element;
+  treeFocus: boolean;
+  layout: "rect" | "radial" | "unrooted" | "clock" | "scatter";
+  dispatch: Dispatch;
+  mobileDisplay: boolean;
+}) {
+  const validLayouts = new Set(["rect", "radial"]);
+  if (!validLayouts.has(layout)) return <></>;
```

Comment:
maybe deserves a little commentary around why the list of valid layouts here is a subset of the allowable types in the method signature?

Reply:
If the concern is that the misalignment is made apparent by the contrasting literal string lists, I'm hoping to change the method signature to `layout: Layout` in the near future by having `Layout` defined somewhere better, for example here in #1864: https://github.com/nextstrain/auspice/blob/b933874b8094ece368655e35a8a08fa3681249b6/src/components/tree/phyloTree/phyloTree.ts#L14

In the meantime, how about adding this as a comment above `const validLayouts`?

> The focus function is only available to layouts that have the concept of a unitless y-axis.

Reply:
> If the concern is that the misalignment is made apparent by the contrasting literal string lists, I'm hoping to change the method signature to `layout: Layout` in the near future by having `Layout` defined somewhere better

It's not the contrasting literal string lists as much as the "if this only works on a subset of layout types why allow all layout types to be passed in?"; IOW even with `layout: Layout` in the method sig, I'd want to know why there's a filtering. 

> In the meantime, how about adding this as a comment above `const validLayouts`?
> 
> > The focus function is only available to layouts that have the concept of a unitless y-axis.

I would say add it, and leave it; it's a good bit of "why" context for the code.